src/components/MDX/MDXComponentFooter.tsx

```diff
@@ -0,0 +1,66 @@
+import { cx } from 'class-variance-authority'
+import { ComponentPropsWithoutRef } from 'react'
+import Link from 'next/link'
+import projectPackage from '../../../package.json'
+
+export interface MDXComponentFooterProps extends ComponentPropsWithoutRef<'div'> {
+  previous: string | null
+  next: string | null
+  filePath: string
+}
+
+export const MDXComponentFooter = ({
```

Comment:
The MDX folder is meant to be used only for MDX components (for example, `H1`, `H2`, etc). The folder `Docs` fits better here IMO. I also think that it makes sense to call it `DocsFooter`

Reply:
Ill move n rename it

Reply:
I had doubts when naming it